aeneas/src/mach/MachLowering.v3

```diff
@@ -1761,8 +1799,11 @@ class MachLowering(mach: MachProgram, compiler: Compiler, config: MachLoweringCo
 				var left = base.inputs[0].dest, right = base.inputs[1].dest;
 				if (SsaConst.?(left)) {
 					// PtrAdd(PtrAdd(#left, right), #offset)
-					var addr = Addr.!(SsaConst.!(left).val);
-					left = context.graph.valConst(pd, addr.add(offset.unbox<int>()));
+					match (SsaConst.!(left).val) {
```

Comment:
Uses of `SsaInstr.unbox` are getting suspicious these days. You might want to use `V3.unboxI32` (and also add a case for `ArrayRangeStart` there too.

Reply:
Should I replace all the `.unbox<int>()`'s in this section (including the old ones)?

Reply:
Yep.